specification/2.0/schema/buffer.schema.json

```diff
@@ -9,6 +9,7 @@
             "type": "string",
             "description": "The uri of the buffer.",
             "format": "uriref",
+            "minLength": 1,
```

Comment:
CC @sbtron 

Our opinion is that this change isn't necessary. As you say an empty string URI points to the glTF file itself (which isn't valid), but you can also put a URI to some other random file (whether glTF or not) and that's also not valid. Clients must deal with this whether or not we allow empty URIs.